pkgs/applications/blockchains/openethereum/default.nix

```diff
@@ -12,16 +12,16 @@
 
 rustPlatform.buildRustPackage.override { stdenv = stdenv; } rec {
```

Comment:
```suggestion
rustPlatform.buildRustPackage rec {
```

Can we potentially rebuilding rust?

Reply:
This was added by @centromere in https://github.com/NixOS/nixpkgs/commit/f8c104d11637fb0489c2b685c2fd32740ee1ca59

Reply:
```suggestion
rustPlatform.buildRustPackage.override { stdenv = stdenv = llvmPackages_12.stdenv; } rec {
```

I would propose to at least do this because otherwise it isn't clear whats happening here.

Reply:
With https://github.com/NixOS/nixpkgs/pull/123074 merged this can be removed.